packages/react-reconciler/src/ReactFiberScheduler.js

```diff
@@ -2123,12 +2125,12 @@ function performWork(minExpirationTime: ExpirationTime, dl: Deadline | null) {
       nextFlushedExpirationTime !== NoWork &&
       (minExpirationTime === NoWork ||
         minExpirationTime >= nextFlushedExpirationTime) &&
-      (!deadlineDidExpire || currentRendererTime >= nextFlushedExpirationTime)
+      (!didYield || currentRendererTime >= nextFlushedExpirationTime)
     ) {
       performWorkOnRoot(
         nextFlushedRoot,
         nextFlushedExpirationTime,
-        currentRendererTime >= nextFlushedExpirationTime,
+        !(currentRendererTime >= nextFlushedExpirationTime),
```

Comment:
Uh, why `! >=` rather than `<`?

Reply:
Planned to change it once I proved it worked, then forgot :D